asm/f32/ddotinc_amd64.s

```diff
@@ -0,0 +1,80 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build !noasm,!appengine
+
+#include "textflag.h"
+
+// func DdotInc(x, y []float32, n, incX, incY, ix, iy int) (sum float64)
+TEXT ·DdotInc(SB), NOSPLIT, $0
+	MOVQ x_base+0(FP), SI  // SI = &x
```

Comment:
I think the convention has become to use `:=` rather than `=`. Can you make this consistent.